src/media-segment-request.js

```diff
@@ -98,6 +98,15 @@ const handleErrors = (error, request) => {
     };
   }
 
+  if (request.responseType === 'arraybuffer' && request.response.byteLength === 0) {
```

Comment:
Added this to `handleErrors` as it was something that was copy pasted for most of our functions.